minifier/minify_examples.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+pushd ../examples
+KEEP_NAMES=1 ./build_all.sh
+popd
+mkdir -p ./out/base/
+for d in ../examples/*/ ; do
+  cp $d/res/*.wasm ./out/base/
+done
+
+#for p in /work/near/core-contracts/*/res/*.wasm ; do
+#  cp $p ./out/base/
+# done
+
+cd out
+for p in ./base/*.wasm ; do
+  w=$(basename -- $p)
+  wasm-snip $p --snip-rust-fmt-code --snip-rust-panicking-code -p core::num::flt2dec::.* -p core::fmt::float::.*  \
+     --output snipped-$w
+  wasm-gc snipped-$w
+  wasm-strip snipped-$w
+  wasm-opt -Oz snipped-$w --output opt-snipped-$w
```

Comment:
Where is this installed for CI?  I had trouble trying to find the best way.

Reply:
Not sure if it's installed on CI.